### PR TITLE
new version of ngs-utils

### PR DESCRIPTION
**Pull request by RoanKanninga**

### CODE REVIEW

File: n/ngs-utils/ngs-utils-23.04.1.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name = 'ngs-utils'
+version = '23.04.1'
+
+homepage = 'https://github.com/molgenis/ngs-utils'
+description = """Collection of notes and scripts related to NGS """
+
+toolchain = {'name': 'system', 'version': ''}
```
Comment (pneerincx) [inline n/ngs-utils/ngs-utils-23.04.1.eb:7]:
```suggestion
toolchain = SYSTEM
```

Comment (pneerincx) [reply n/ngs-utils/ngs-utils-23.04.1.eb:7]: Both will work for now, but the latter is the future way to go.

**Review by pneerincx — CHANGES_REQUESTED** [review]
See inline comment.

File: n/ngs-utils/ngs-utils-23.04.1.eb (hunk 1 above)
Comment (RoanKanninga) [reply n/ngs-utils/ngs-utils-23.04.1.eb]: okay, check

File: n/ngs-utils/ngs-utils-23.04.1.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name = 'ngs-utils'
+version = '23.04.1'
+
+homepage = 'https://github.com/molgenis/ngs-utils'
+description = """Collection of notes and scripts related to NGS """
+
+toolchain = {'name': 'SYSTEM', 'version': ''}
```
Comment (pneerincx) [inline n/ngs-utils/ngs-utils-23.04.1.eb:7]:
```suggestion
toolchain = SYSTEM
```

Comment (pneerincx) [reply n/ngs-utils/ngs-utils-23.04.1.eb:7]: You previously changed it in way that won't work. `SYSTEM` is a constant.

**Review by pneerincx — CHANGES_REQUESTED** [review]
See comment.

**Review by pneerincx — APPROVED** [review]

**Pull request merged**